### PR TITLE
Improve menus listview 2

### PR DESCRIPTION
**Pull request by chmst**
Repeat https://github.com/joomla/joomla-cms/pull/22112, because the branch could not be updated

### CODE REVIEW

File: administrator/components/com_menus/tmpl/menus/default.php [hunk 1]
```diff
@@ -54,12 +54,15 @@
 					<table class="table" id="menuList">
 						<thead>
 							<tr>
-								<td style="width:1%" class="text-center">
+								<td style="width:1%">
```
Comment (brianteeman) [inline administrator/components/com_menus/tmpl/menus/default.php:57]: please put the class="text-center" back

**Review by brianteeman — APPROVED** [review]
approve language changes

**Comment by wilsonge** [comment]
LGTM :)

**Comment by brianteeman** [comment]
thanks - it was a bit quick merging - I will do another pr later

File: administrator/components/com_menus/tmpl/menus/default.php (hunk 1 above)
Comment (chmst) [reply administrator/components/com_menus/tmpl/menus/default.php:57]: too late! A new PR for you :)

**Comment by brianteeman** [comment]
see #22146 for small update

**Pull request merged**